packages/roosterjs-content-model-plugins/lib/shortcut/shortcuts.ts

```diff
@@ -129,17 +129,16 @@ export const ShortcutRedo: ShortcutCommand = {
 
 /**
  * Shortcut command for Redo 2
- * Windows: N/A
+ * Windows: Ctrl + Shift + Z
  * MacOS: Meta + Shift + Z
  */
-export const ShortcutRedoMacOS: ShortcutCommand = {
+export const ShortcutRedoAlt: ShortcutCommand = {
```

Comment:
This makes it a breaking change.

Can you do this:

export const ShortcutRedoAlt = {.... }

/**
 * @deprecated
 */
const export ShortcutRedoMacOS = ShortcutRedoAlt;

So that old one still work, and new one also has better name